pset/utils.go

```diff
@@ -371,3 +371,19 @@ func validateSignature(sig []byte) bool {
 func checkValid(ps psbt.PartialSig) bool {
 	return validatePubkey(ps.PubKey) && validateSignature(ps.Signature)
 }
+
+// reverseBytes returns the given byte slice with elems in reverse order.
+func reverseBytes(buf []byte) []byte {
```

Comment:
Why not `sort.Reverse` method?

Reply:
The functions you mentioned work with signatures,  to eventually change the order of them. The goal of `reverseBytes` function, instead, is to reverse a byte slice. This is necessary when encoding the transaction hash. I don't understand the correlation between this helper and the ones of the partial signature sorter.